examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+```
+
+Now we will need to make a service for the workers HTTP server just like for the coordinator. For this, we will be using the port 9000. Again, if necessary, you can configure the NodePort as well.
+(note: we are using LoadBalancer, because we will need to expose that for the public)
+(won`t a LB kill the purpose of a ping-check though?)
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: worker-lb-svc
+  namespace: cloudretro
+spec:
+  selector:
+    app: worker
+  ports:
+    - port: 9000
+      targetPort: 9000
+      #nodePort: 30002
+  type: LoadBalancer
+EOF
+```
+
+Another k8s service will be needed for the UDP service, although this time we will use a ClusterIP. Because of STUNner, we will be using the in-cluster network for that.
+For the UDP port, you should be using the one you`ve used at the worker deployment. NodePort is not needed here, because we are not planning to expose this service.
+
+```console
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: worker-ci-udp-svc
+  namespace: cloudretro
+spec:
+  selector:
+    app: worker
+  ports:
+    - protocol: UDP
+      port: 8443
+      targetPort: 8443
+  type: ClusterIP
+EOF
+```
+
+Now, checking up on services should be looking like this:
+
+```console
+kubectl get services -n cloudretro
+
+NAME                 TYPE           CLUSTER-IP            EXTERNAL-IP             PORT(S)          AGE
+coordinator-lb-svc   LoadBalancer   <YOUR_C_CLUSTER_IP>   <YOUR_C_EXTERNAL_IP>    8000:30618/TCP   18h
+worker-ci-udp-svc    ClusterIP      10.104.129.8          <none>                  8443/UDP         3m2s
+worker-lb-svc        LoadBalancer   <YOUR_W_CLUSTER_IP>   <YOUR_W_EXTERNAL_IP>    9000:32681/TCP   3m10s
+```
+
+Note down <YOUR_C_CLUSTER_IP>, <YOUR_C_EXTERNAL_IP>, <YOUR_W_EXTERNAL_IP>. For the next configuration steps, we will need these.
+
+#### CloudRetro config once again
+
+Fortunately we`ve made our configmap in the first step, so our lives are a little easier now.
+We are going to config up CloudRetro with these values:
+
+* <YOUR_C_CLUSTER_IP>  - The in-cluster IP of your coordinator, the workers should connect to this.
+* <YOUR_C_EXTERNAL_IP> - The exposed IP of your coordinator, clients should use this to access the service at the sepcified port (8000 in this example).
+* <YOUR_W_EXTERNAL_IP> - The exposed IP of the collection of your workers.
+
+To use these values, we need to edit the configmap weve made:
+
+```console
+kubectl edit configmap -n cloudretro cloudretro-config
+```
+
+Then, we should be presented an ugly looking config. Finding the proper attributes and overwriting their values is the next step.
+You can find the nice looking version of it [here](https://github.com/giongto35/cloud-game/blob/master/configs/config.yaml), and their line numbers presented as ln.
+
+* (ln. 41) coordinator -> server -> address: Should be the port youve used during the coordinator deployment (8000 in this example)
```

Comment:
These should be specified in a different section (maybe something like "process flow" or similar): in this section we are interested in the setup instructions, and not necessarily with the way how information flows through the system